Automattic/wp-calypso
# Update/import done link to checklist  in progress stage

#### Changes proposed in this Pull Request

This is a follow up to https://github.com/Automattic/wp-calypso/pull/28515 which adds a link to the checklist from the 'done' stage of the importer. This attempts to apply the same treatment to the 'in-progress' stage.
Personally, this feels a bit of a mouthful here though, so we might want to revise the message and what is / isn't included in it:

<img width="728" alt="screen shot 2018-11-14 at 13 20 12" src="https://user-images.githubusercontent.com/4335450/48485539-0faec480-e811-11e8-9149-81c6ea8dd570.png">

#### Testing instructions

* Go to http://calypso.localhost:3000/settings/import
* Start a WordPress import and follow the instructions
  * After starting the import and mapping authors, you'll see a progress bar with copy that says: "Importing may take a while if your site has a lot of media, but you can safely navigate away from this page if you need to or go to our `handy checklist` to explore your settings and customizing options. we'll send you a notification when it's done."
  * Clicking the "handy checklist" link should take you to your site checklist.
